topics/assembly/tutorials/flye-assembly/tutorial.md

```diff
@@ -0,0 +1,253 @@
+---
+layout: tutorial_hands_on
+
+title: 'Genome assembly using PacBio data'
+zenodo_link: 'https://zenodo.org/record/5702408#.YZUb5uvjIiU'
+tags:
+  - assembly
+  - pacbio
+questions:
+- How to perform genome assembly with PacBio data ?
+- How to check assembly quality ?
+objectives:
+- Assemble a Genome
+- Assess assembly quality
+time_estimation: 6h
+level: Intermediate
+key_points:
+- The take-home messages
+- They will appear at the end of the tutorial
+contributors:
+- abretaud
+- alexcorm
+- r1corre
+- lleroi
+- stephanierobin
+
+---
+
+
+# Introduction
+{:.no_toc}
+
+In this tutorial, we will assemble a genome of a species of fungi in the family Mucoraceae, *Mucor mucedo*, from PacBio sequencing data. These data were obtained from NCBI (SRR8534473 SRR8534474 SRR8534475). The quality of the assembly obtained will be analyzed, in particular by comparing it to a reference assembly, obtained with Falcon assembler, and available on [JGI website](https://mycocosm.jgi.doe.gov/Mucmuc1/Mucmuc1.info.html).
```

Comment:
Explain that it's HiFi data (it is?)

Reply:
No because it's not Hifi data, these data are CLR data, I will clarify that